content/en/policy-controller/overview.md

```diff
@@ -321,10 +321,50 @@ You can find more information about these fields in [here](https://github.com/si
 spec:
   authorities:
     - keyless:
-        url: https://fulcio.example.com
-      identities:
-        - issuer: 'https://issuer/'
-          subject: 'foo@example.dev'
+        url: https://fulcio.sigstore.dev
+        identities:
+          - issuer: 'https://issuer/'
+            subject: 'foo@example.dev'
+      rfc3161timestamp:
+        trustRootRef: my-tsa-keys
+```
+
+In case of using a different sigstore instance, you should specify the certificate chain of this new certificate authority in the `TrustRoot`:
```

Comment:
I think I'd prefer having this section below that's explicitly for different sigstore instances? I think otherwise it's going to get unwieldy having it in various places? WDYT?

Reply:
Change of plans! I'll remove this section as it doesn't add much value in comparison with the section below.